### PR TITLE
New config for NIC bonding

### PR DESCRIPTION
**Pull request by weihanglo**
Revive #22. The configurations of equinix and general need update as well.

### CODE REVIEW

**Review by guangbochen — APPROVED** [review]
LGTM

File: equinix/README.md [hunk 1]
```diff
@@ -47,7 +47,13 @@ The `harvester.install.config_url=https://metadata.platformequinix.com/userdata`
       password: p@ssword  # replace with a your password
     install:
       mode: create
-      mgmt_interface: eth0
+      networks:
+        harvester-mgmt: # The management bond name. This is mandatory.
+          bond_option:
+            mode: balance-tlb
```
Comment (guangyee) [inline equinix/README.md:53]: Is `mode: balance-tlb` mandatory? If not, will `balance-tlb` be the default mode if absent?

File: equinix/README.md [hunk 2]
```diff
@@ -47,7 +47,13 @@ The `harvester.install.config_url=https://metadata.platformequinix.com/userdata`
...
+          interfaces:
+          - name: eth0
```
Comment (guangyee) [inline equinix/README.md:55]: Do we also need to specify `default_route: true` here to establish the default route? If not, is default route assumed if absent?

File: equinix/README.md (hunk 1 above)
Comment (weihanglo) [reply equinix/README.md:53]: Default to `balance-tlb` if missing. I am fine whether put it here or remove it.

File: equinix/README.md (hunk 2 above)
Comment (weihanglo) [reply equinix/README.md:55]: `harvester-mgmt` becomes the default route if none of networks there has a `default_route: true`. It is great to add `default_route` for the purpose of an example. WDYT?

File: equinix/README.md (hunk 1 above)
Comment (guangyee) [reply equinix/README.md:53]: We can leave it here. I was just curious. Though generally speaking, configs that has default values does not need to be illustrated in the examples unless they are relevant to the situation.

File: equinix/README.md (hunk 2 above)
Comment (guangyee) [reply equinix/README.md:55]: Yes, I think we can use that to illustrate the choice for default route.

**Comment by weihanglo** [comment]
All issues addressed. Thanks @guangyee for your review.

**Review by guangyee — APPROVED** [review]
awesome, thanks!

**Pull request merged**